homeassistant/components/airpatrol/quality_scale.yaml

```diff
@@ -0,0 +1,64 @@
+rules:
+  # Bronze
+  action-setup:
+    status: exempt
+    comment: Integration does not register custom actions.
+  appropriate-polling: done
+  brands:
+    status: exempt
+    comment: Basic integration that does not need to be in the brands registry.
```

Comment:
This is not a basic integration :)

Reply:
Matter of opinion maybe ;)
No idea what basic means but yes, needs a brand PR